src/ReactiveDomain.Persistence/EventStore/EventStoreConnectionWrapper.cs

```diff
@@ -17,8 +17,8 @@ public EventStoreConnectionWrapper(ES.IEventStoreConnection eventStoreConnection
         {
             Ensure.NotNull(eventStoreConnection, nameof(eventStoreConnection));
             EsConnection = eventStoreConnection;
-            EsConnection.Connected += ConnOnConnected;
-            EsConnection.Disconnected += ConnOnDisconnected;
+            //EsConnection.Connected += ConnOnConnected;
+            //EsConnection.Disconnected += ConnOnDisconnected;
```

Comment:
Is this intentional? If so, can we delete `ConnOnConnected` and `ConnOnDisconnected`?

Reply:
There are issues around these events with different versions of ES. They are used to allow the system to know if the connection has been dropped so I'd like to keep them if possible. Needs more research, and there are a bunch of other things we should change if removing this permanently. I think this is an issue beyond the scope of what we are doing with this PR.
Added a TODO